board/safety/safety_ford.h

```diff
@@ -119,7 +119,7 @@ static int ford_tx_hook(CANPacket_t *to_send) {
     bool steer_control_enabled = steer_control_type != 0U;
 
     // No steer control allowed when controls are not allowed
-    if (!controls_allowed && steer_control_enabled) {
+    if (!get_lateral_allowed() && steer_control_enabled) {
```

Comment:
@incognitojam this is the last port to have custom lateral checks. can we do something similar to tesla and nissan, but for curvature rate limiting?